java/core/src/test/java/com/google/protobuf/UnknownFieldSetPerformanceTest.java

```diff
@@ -0,0 +1,78 @@
+// Protocol Buffers - Google's data interchange format
+// Copyright 2008 Google Inc.  All rights reserved.
```

Comment:
2021 and otherwise update to current guidelines for copyright statements

Reply:
```suggestion
// Copyright 2008 Google Inc.  All rights reserved.bnnjjj
```